buildtest/cli/__init__.py

```diff
@@ -82,6 +83,35 @@ def positive_number(value):
     return int_val
 
 
+def supported_color(value):
```

Comment:
you might want to use a meaningful argument instead of `value` perhaps say `input_color` or `color_choice`

Reply:
done